client/src/components/geneExpression/infoPanel/index.tsx

```diff
@@ -0,0 +1,104 @@
+import React from "react";
+import { connect } from "react-redux";
+import { AnchorButton, ButtonGroup } from "@blueprintjs/core";
+import { AppDispatch, RootState } from "../../../reducers";
+import {
+  CollapseToggle,
+  InfoPanelContent,
+  InfoPanelHeader,
+  InfoPanelTabs,
+  InfoPanelWrapper,
+  StyledAnchorButton,
+} from "./style";
+import GeneInfo from "./geneInfo";
+import DatasetInfo from "./datasetInfo";
+
+interface Props {
```

Comment:
Non-blocking: Maybe as a post launch P0 - worth refactoring types out to its own file like we do in Data Portal 🙆‍♂️ ?